sample-apps/react/react-dogfood/helpers/jwt.ts

```diff
@@ -9,34 +10,29 @@ export const maxTokenValidityInSeconds: number =
 
 export const createToken = (
   userId: string,
+  apiKey: string,
   jwtSecret: string,
   params: Record<string, string | string[]> = {},
 ) => {
+  const client = new StreamClient(apiKey, jwtSecret);
+
   const {
     exp, // expiration, in seconds from now
     ...rest
   } = params;
 
-  const expiryFromNowInSeconds = exp
-    ? parseInt(exp as string, 10)
-    : maxTokenValidityInSeconds;
-  const expiration = Math.round(
-    Date.now() / 1000 +
-      Math.min(expiryFromNowInSeconds, maxTokenValidityInSeconds),
-  );
-
-  const payload: Record<string, unknown> = {
+  return client.generateUserToken({
     iss: 'https://pronto.getstream.io',
     sub: `user/${userId}`,
     // subtract 5 seconds, sometimes the coordinator fails with
     // "token used before issued at (iat)" error
     iat: Math.round(Date.now() / 1000) - 5,
     ...rest,
     user_id: userId,
-    exp: expiration,
```

Comment:
I think `exp` is the JWT standard field for token validity check. Is this something that has changed on our backend too?

Reply:
No, but the node-sdk will set `exp` from `validity_in_seconds`, this is there to make token generator easier for integrators, no need to deal with timestamps

Reply:
ah, I see! Thanks!